osbs/build/plugins_configuration.py

```diff
@@ -174,68 +174,6 @@ def render(self):
         """
         raise NotImplementedError
 
-    def has_tag_suffixes_placeholder(self):
-        phase = 'postbuild_plugins'
-        plugin = 'tag_from_config'
-        if not self.pt.has_plugin_conf(phase, plugin):
-            logger.debug('no tag suffix placeholder')
-            return False
-
-        placeholder = '{{TAG_SUFFIXES}}'
-        plugin_conf = self.pt.get_plugin_conf('postbuild_plugins', 'tag_from_config')
-        return plugin_conf.get('args', {}).get('tag_suffixes') == placeholder
```

Comment:
Why is this being removed without adding an atomic reactor counterpart? Should it be in a different commit?

Reply:
we don't need this, that placeholder was always in json inner template, and this wasn't providing any value, IF you don't want to plugin to run, you can just simply remove it from template

Reply:
This seems to be unrelated to the commit that is moving code to atomic-reactor. Could it be in a separate commit? Alternatively, you could add a longer description to the commit message (e.g. bullets) explaining this specific change and the removal of `stop_autorebuild_disabled` saying both are a consequence of this change.

Reply:
updated commit msg